### PR TITLE
Qr code plugin

### PR DESCRIPTION
**Pull request by psykomantis**
Qr plugin, given a text it returns a qr code (with colors if you want)

### CODE REVIEW

File: plugins/qr.lua [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  local url = "http://api.qrserver.com/v1/create-qr-code/?" .. query .. "&size=600x600"  --fixed size otherways it's low detailed
+  local response, code, headers = http.request(url)
+
+  if code ~= 200 then
+    return "Oops! Error: " .. code
+  end
+
+  if #response > 0 then
+	   send_photo_from_url(receiver, url)
+	return
```
Comment (Walkman100) [inline plugins/qr.lua]: Indentation screw up :worried:

Comment (psykomantis) [inline plugins/qr.lua]: Which identation walkman? I don't understand !

Comment (Walkman100) [inline plugins/qr.lua]: the line above the one I commented on. you have:
```lua
  if #response > 0 then
	   send_photo_from_url(receiver, url)
	return

  end
  return "Oops! Something strange happened :("
```

Comment (psykomantis) [inline plugins/qr.lua]: Oh it's true :/

**Pull request merged**